src/features/wallet/send/SendTransferForm.tsx

```diff
@@ -0,0 +1,420 @@
+  const renderLockedPaymentForm = () => [
+    <LockedField label={t('send.address.label')} value={address} />,
+    <LockedField
+      label={t('send.amount.label')}
+      value={amount}
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+      bottom
+    />,
+  ]
+
+  const renderLockedBurnForm = () => [
+    <LockedField label={t('send.address.label')} value={address} />,
+    <LockedField
+      label={t('send.amount.label')}
+      value={amount}
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+    />,
+    <LockedField label={t('send.dcAmount.label')} value={dcAmount} />,
+    <LockedField label={t('send.memo.label')} value={memo} bottom />,
+  ]
+
+  const renderPaymentForm = () => [
+    <InputField
+      defaultValue={address}
+      onChange={setAddress}
+      label={t('send.address.label')}
+      placeholder={t('send.address.placeholder')}
+      extra={
+        <AddressExtra
+          addressLoading={addressLoading}
+          isValidAddress={Address.isValid(address)}
+          onScanPress={onScanPress}
+        />
+      }
+      footer={<AddressAliasFooter addressAlias={addressAlias} />}
+    />,
+    <InputField
+      type="numeric"
+      defaultValue={amount}
+      onChange={setFormAmount}
+      value={amount}
+      label={t('send.amount.label')}
+      placeholder={t('send.amount.placeholder')}
+      extra={
+        <TouchableOpacityBox onPress={setMaxAmount}>
+          <Text fontSize={12} color="primaryMain">
+            {t('send.sendMax')}
+          </Text>
+        </TouchableOpacityBox>
+      }
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+    />,
+  ]
+
+  const renderBurnForm = () => [
+    <InputField
+      defaultValue={address}
+      onChange={setAddress}
+      label={t('send.address.label')}
+      placeholder={t('send.address.placeholder')}
+      extra={
+        Address.isValid(address) ? (
+          <Box padding="s" position="absolute" right={0}>
+            <Check />
+          </Box>
+        ) : (
+          <TouchableOpacityBox
+            onPress={onScanPress}
+            padding="s"
+            position="absolute"
+            right={0}
+          >
+            <QrCode width={16} color={primaryMain} />
+          </TouchableOpacityBox>
+        )
+      }
+    />,
+    <InputField
+      type="numeric"
+      defaultValue={amount}
+      onChange={setAmount}
+      value={amount}
+      label={t('send.amount.label')}
+      placeholder={t('send.amount.placeholder')}
+      footer={amount ? <FeeFooter fee={fee} /> : undefined}
+    />,
+    <InputField
+      type="numeric"
+      defaultValue={dcAmount}
+      onChange={setDcAmount}
+      label={t('send.dcAmount.label')}
+      placeholder={t('send.dcAmount.placeholder')}
+    />,
+    <InputField
+      defaultValue={memo}
+      onChange={setMemo}
+      label={t('send.memo.label')}
+      placeholder={t('send.memo.placeholder')}
+    />,
+  ]
+
+  const renderSellerTransferForm = () => [
```

Comment:
This shouldn't return an array, it should use `(` and `)` instead of `[` and `]`